sklearn/utils/tests/test_estimator_html_repr.py

```diff
@@ -140,6 +140,12 @@ def test_get_visual_block_column_transformer():
     assert est_html_info.name_details == (["num1", "num2"], [0, 3])
 
 
+def test_estimator_html_repr_an_empty_pipeline():
+    empty_pipeline = Pipeline([])
+    # Smoke test
```

Comment:
We don't need to specify smoke test here since the description of the test is already informative.

```suggestion
```